### DISCUSSION TITLE
can we hire a php developer here?

### DISCUSSION THREAD

**Discussion by ankit230312** [community/community#52226] [New to GitHub]
php Framework expert

**Comment by queenofcorgis** [comment]
Hi @ankit230312 , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
